gui/choc_WebView.h

```diff
@@ -135,7 +135,8 @@ class WebView
     void navigate (const std::string& url);
 
     /// A callback function which can be passed to bind().
-    using CallbackFn = std::function<choc::value::Value(const choc::value::ValueView& args)>;
+    using ReplyFn = std::function<void(choc::value::Value, std::string)>;
+    using CallbackFn = std::function<void(const choc::value::ValueView& args, ReplyFn&& reply)>;
```

Comment:
clarifying question: in this new world, all functions would be required to call `reply` (at their leisure), and not be required to return anything. is that right?

Reply:
Right!